install/src/main/scripts/post-install.sh

```diff
@@ -20,6 +20,19 @@
 # #L%
 ###
 
+# function for determining way to handle startup scripts
+function get_linux_type {
+# redhat
+which chkconfig > /dev/null && echo "chkonfig" && return 0
+# ubuntu sysv
+which update-rc.d > /dev/null && echo "update-rc.d" && return 0
+echo "Couldn't recognize linux version, exiting..."
```

Comment:
Could the default be "manual" instead of exiting? The user could manually execute our run scripts.